src/containers/Home/Home.tsx

```diff
@@ -24,7 +24,7 @@ const Home: FC<IAppProps> = props => {
 
   useEffect(() => {
     async function onLoad() {
```

Comment:
Why create a new method every time `props.checkedAuth`?

Reply:
The fixed for this is in a different branch, this was the legacy code of the base forked from the serverless-stack-client.

Reply:
In this branch, I only converted files regarding auth and nothing else.

Reply:
👍